src/pot.sol

```diff
@@ -123,7 +123,7 @@ contract Pot is DSNote {
 
     // --- Savings Rate Accumulation ---
     function drip() external note {
-        require(now >= rho);
+        require(now >= rho, "no-double-drip");
```

Comment:
You might want to leave the messages empty if this is going to be used as a lemma in FV somewhere. If you still want to keep them then please prefix with `dss/`.